general/server/media/shows.md

```diff
@@ -13,6 +13,10 @@ The most common naming scheme for shows is categorizing the files by series and
         /Season 01
             Episode S01E01.mkv
             Episode S01E02.mkv
+        /Season 02
+            Episode S02E01.mkv
+            Episode S02E02.mkv
+            
```

Comment:
not sure there should be a newline here

Reply:
That newline, separate the "show 1" of "show 2", but seeing the original doc, they were joined. So yes, there should not be a new line